tests/ccerts.c

```diff
@@ -0,0 +1,84 @@
+/* Copyright (C) 2024 Simo Sorce <simo@redhat.com>
+   SPDX-License-Identifier: Apache-2.0 */
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <openssl/ssl.h>
```

Comment:
is the `ssl.h` header used in this test? And `rsa.h`? Or is it just a leftover from some of the previous tests?

Reply:
copy/paste :-)